StyleCop.Analyzers/StyleCop.Analyzers/NamingRules/SA1304NonPrivateReadonlyFieldsMustBeginWithUpperCaseLetter.cs

```diff
@@ -45,6 +45,9 @@ internal class SA1304NonPrivateReadonlyFieldsMustBeginWithUpperCaseLetter : Diag
 
         private static readonly Action<SyntaxNodeAnalysisContext> FieldDeclarationAction = HandleFieldDeclaration;
 
+        private static readonly DiagnosticDescriptor SA1307Descriptor =
+            new SA1307AccessibleFieldsMustBeginWithUpperCaseLetter().SupportedDiagnostics.Single();
+
```

Comment:
❓ Wouldn't it be a better option to make the (already static) `Descriptor` member of SA1307 `internal` and use that instead?

Reply:
It made a mess of the ordering rules. But I suppose I can do that and suppress the diagnostic it causes 😄

Reply:
@vweijsters This is now fixed 😄